history.adoc

```diff
@@ -1,12 +1,13 @@
 :issues: https://github.com/cf-convention/cf-conventions/issues/
 :pull-requests: https://github.com/cf-convention/cf-conventions/pull/
-:tickets: http://cfconventions.org/Data/Trac-tickets/
+:tickets: https://cfconventions.org/Data/Trac-tickets/
 
 [[revhistory, Revision History]]
 == Revision History
 
 === Working version (most recent first)
 
+* {issues}463[Issue #463]: Convert all referenced URLs with HTTP protocol to HTTPS
```

Comment:
Thanks Lars, that seems like a great update to reflect the updates. I'll convert to that in a new commit.

Reply:
Updated in [cc7ad26](https://github.com/cf-convention/cf-conventions/pull/476/commits/cc7ad26612f2388afbb582f261030179f2d55c36). I added you as a co-author, Lars, since you provided the text suggestion so deserve credit.

Marking as resolved.